tests/serialization/serializers/test_serializers.py

```diff
@@ -26,19 +27,22 @@
 import pytest
 from dateutil.tz import tzutc
 from deltalake import DeltaTable
+from packaging.version import Version
 from pendulum import DateTime
 from pendulum.tz.timezone import FixedTimezone, Timezone
 
-from airflow import PY39
 from airflow.models.param import Param, ParamsDict
 from airflow.serialization.serde import DATA, deserialize, serialize
 
-if PY39:
+if sys.version_info >= (3, 9):
```

Comment:
The python version check is unnecessary. I know this because my own changes, that don't include this guard, passed the tests (that ran on py 3.8 as far as I can tell) and got merged.

Reply:
The API of importlib.metadata not fully compatible between provisional (3.8-3.9) an stable (3.10+)
I think maybe it should be compatible in versions, since we use it in 3.9 which also provisional.

My thoughts that we should use bakported version, just not for thinking about is this functional compatible or not.
But since this part about the tests we might use only stdlib version

Reply:
On the one hand, we're not using whichever part of the API of `importlib.metadata` that changed when transitioning from provisional to stable (so in that sense, provisional is fine) here. On the other hand, as you mentioned, 3.9 is also still provisional, so there's no specific reason for checking for _that_. 

So yeah, I prefer this check gone, but if you feel it's needed, I think you should check for 3.10 instead.

Reply:
> But since this part about the tests we might use only stdlib version